backend/muster-roll-service/src/main/java/org/egov/service/MusterRollService.java

```diff
@@ -121,18 +122,22 @@ public List<MusterRoll> searchMusterRolls(RequestInfoWrapper requestInfoWrapper,
 
         musterRollValidator.validateSearchMuster(requestInfoWrapper,searchCriteria);
         enrichmentService.enrichSearchRequest(searchCriteria);
-        List<MusterRoll> musterRollList = musterRollRepository.getMusterRoll(searchCriteria);
-        List<MusterRoll> filteredMusterRollList = musterRollList;
+
         List<Role> roles = requestInfoWrapper.getRequestInfo().getUserInfo().getRoles();
         boolean isFilterRequired = roles.stream()
                 .anyMatch(role -> role.getCode().equalsIgnoreCase("ORG_ADMIN") || role.getCode().equalsIgnoreCase("ORG_STAFF"));
```

Comment:
@anitha-srinivasan-egov - the roles getting hardcoded in the code is not a good pattern. Vishal has made it configurable.

Reply:
As discussed we have parked it for now. We will decide on the approach after further discussion